csi/src/node/iscsiutil.rs

```diff
@@ -39,13 +39,18 @@ fn get_iscsiadm() -> Result<&'static str, Error> {
     }
 }
 
-fn wait_for_path_to_exist(devpath: &str, max_retries: i32) -> bool {
-    let second = time::Duration::from_millis(1000);
+fn wait_for_path_to_exist(
+    devpath: &str,
+    sleepmillis: u64,
+    millis: u64,
```

Comment:
Why not something like this:

```
fn wait_for_for_path(devpath: &str, timeout: Duration, retires: u32) {... } 
```

Reply:
Done. https://github.com/openebs/Mayastor/pull/320/commits/d9ab94c314e28a79663c8d9be0921f8e12834a77